docs/enterprise/about.md

```diff
@@ -0,0 +1,53 @@
+---
+title: About
+sidebarDepth: 1
+description: What does the Pomerium Enterpise Console offer?
+---
+
+# Pomerium Enterprise
+
+This paragraph introduces what Pomerium Enterprise is.
+
+## Features
+
+In addition to the capabilities provided by open-source Pomerium, Pomerium Enterprise provides:
+
+### Management GUI
+
+The Pomerium Enterprise Console lets you view traffic and logs, define routes and policies, and organize your service access all from an intuitive web interface.
+
+![Overview animation of the Pomerium Enterprise Console](./img/console-overview.gif)
+
+### Programmatic API
+
+<!-- @Travis please add details. -->
+
+### Session management
+
+Quickly view who is logged in your infrastructure, with easy access to revoke sessions.
+
+![Pomerium Enterprise Console Session List](./img/console-session-list.png)
+
+### Self-Service, and Access Controls
+
+Easily define who can control access to what areas of your infrastructure. Our Namespaces make it easy to allow teams to self-manage access to the infrastructure they build from or depend on.
+
+Pomerium Enterprise uses teams and groups defined by your IdP, so you can build stable policies that's don't need to be adjusted as your company changes.
+
+### Deployment History & Audit Logs
+
+View and export change and access logs from the web UI. Pomerium Enterprise Console gives you a complete view of who's using it and how access is adjusted.
```

Comment:
This PRD gives a really good background.
https://www.notion.so/pomerium/Decision-audit-logging-427efa9727244a468303245896672fa5